internal/controller/api/v1beta1/infraenv_types.go

```diff
@@ -108,6 +108,7 @@ type InfraEnvStatus struct {
 	// ISODownloadURL specifies an HTTP/S URL that contains a discovery ISO containing the
 	// configuration from this InfraEnv.
 	ISODownloadURL string                   `json:"isoDownloadURL,omitempty"`
+	CreatedAt      *metav1.Time             `json:"isoCreatedAt,omitempty"`
```

Comment:
Not sure hat we need a different name between json and code,

Reply:
Changed to `createdAt`